src/shared/avl/utils.ts

```diff
@@ -18,7 +18,7 @@ import { SiriSchemaTransformed } from "../schema";
 import { AvlSubscription, avlSubscriptionSchema, avlSubscriptionsSchema } from "../schema/avl-subscribe.schema";
 import { vehicleActivitySchema } from "../schema/avl.schema";
 import { chunkArray } from "../utils";
-import { NM_TOKEN_DISALLOWED_CHARS_REGEX, SIRI_VM_POPULATED_STRING_TYPE_DISALLOWED_CHARS_REGEX } from "../validation";
+import {} from "../validation";
```

Comment:
Can this be removed?

Reply:
interestingly biome is supposed to do this but looks like an open issue: https://github.com/biomejs/biome/discussions/2646